tetragonpod/Makefile

```diff
@@ -0,0 +1,129 @@
+
```

Comment:
are we using any target in this makefile?

Reply:
ok then let's remove it.

Reply:
sure